asapdiscovery/docking/scripts/prep_proteins.py

```diff
@@ -122,6 +123,11 @@ def prep_mp(
             initial_prot, res_list, xtal.protein_chains
         )
 
+    ## Delete extra copies of ligand in the complex
+    initial_prot = remove_extra_ligands(
+        initial_prot, lig_chain=xtal.active_site_chain
+    )
+
```

Comment:
i think in the future we'll need to add this as an option or something for the other structures but that's for a future PR to fix